tests/conftest.py

```diff
@@ -266,12 +268,19 @@ def small_array_filters(small_array: np.ndarray, load_scan: LoadScan, filter_con
 # IO fixtures
 @pytest.fixture
 def load_scan(loading_config: dict) -> LoadScan:
-    """Instantiate a LoadScan object."""
+    """Instantiate a LoadScan object from a .spm file."""
     scan_loader = LoadScan(RESOURCES / "minicircle.spm", **loading_config)
     return scan_loader
 
 
 @pytest.fixture
+def load_scan_ibw() -> LoadScan:
+    """Instantiate a LoadScan object from a .ibw file."""
+    scan_loader = LoadScan(RESOURCES / "minicircle2.ibw", channel = "HeightTracee")
+    return scan_loader
+
+
+@pytest.fixture # I'm not sure this is used?
```

Comment:
It's used in the `test_plottingfuncs.py`...

```
❱ grep load_scan_data tests/*
tests/conftest.py:def load_scan_data(load_scan: LoadScan) -> LoadScan:
grep: tests/output: Is a directory
grep: tests/processed_minicircle: Is a directory
grep: tests/__pycache__: Is a directory
grep: tests/_regtest_outputs: Is a directory
grep: tests/resources: Is a directory
tests/test_plottingfuncs.py:def test_plot_and_save_no_colorbar(load_scan_data: LoadScan, tmp_path: Path) -> None:
tests/test_plottingfuncs.py:        data=load_scan_data.image,
tests/test_plottingfuncs.py:        pixel_to_nm_scaling_factor=load_scan_data.pixel_to_nm_scaling,
tests/test_plottingfuncs.py:def test_plot_and_save_colorbar(load_scan_data: LoadScan, tmp_path: Path) -> None:
tests/test_plottingfuncs.py:        data=load_scan_data.image,
tests/test_plottingfuncs.py:        pixel_to_nm_scaling_factor=load_scan_data.pixel_to_nm_scaling,
tests/test_plottingfuncs.py:def test_plot_and_save_no_axes(load_scan_data: LoadScan, plotting_config: dict, tmp_path: Path) -> None:
tests/test_plottingfuncs.py:        data=load_scan_data.image,
tests/test_plottingfuncs.py:def test_plot_and_save_no_axes_no_colorbar(load_scan_data: LoadScan, plotting_config: dict, tmp_path: Path) -> None:
tests/test_plottingfuncs.py:        data=load_scan_data.image,
tests/test_plottingfuncs.py:def test_plot_and_save_colorbar_afmhot(load_scan_data: LoadScan, tmp_path: Path, plotting_config: dict) -> None:
tests/test_plottingfuncs.py:        data=load_scan_data.image,
tests/test_plottingfuncs.py:        pixel_to_nm_scaling_factor=load_scan_data.pixel_to_nm_scaling,
grep: tests/tracing: Is a directory
```

Reply:
Cool, I'll delete my comment